### PR TITLE
Fix two bounds control related issues

### PR DESCRIPTION
**Pull request by MaxWang-MS**
## Overview
This PR fixes the issue of the translation handles of bounds control having wrong rotation and the issue of some slates cannot be scaled with near interaction.

## Changes
- Fixes: #9334 and #9346.

## Verification
Please see the issues above for steps to verify these fixes.

### CODE REVIEW

**Comment by davidkline-ms** [comment]
This looks reasonable to me. I would appreciate if @cre8ivepark could verify and @CDiaz-MS could review.

File: Assets/MRTK/SDK/Features/UX/Scripts/BoundsControl/Visuals/TranslationHandles.cs [hunk 1]
```diff
@@ -46,11 +46,11 @@ protected override Quaternion GetRotationRealignment(int handleIndex)
             // Align handle with its edge assuming that the prefab is initially aligned with the up direction 
             if (VisualUtils.FaceAxisType[handleIndex] == CardinalAxisType.X)
             {
-                return Quaternion.FromToRotation(Vector3.up, directionSign * Vector3.right);
+                return Quaternion.FromToRotation(Vector3.forward, directionSign * Vector3.right);
```
Comment (CDiaz-MS) [inline Assets/MRTK/SDK/Features/UX/Scripts/BoundsControl/Visuals/TranslationHandles.cs:49]: Was the handle rotation issue a regression or has this rotation always been in the wrong direction?

Comment (CDiaz-MS) [reply Assets/MRTK/SDK/Features/UX/Scripts/BoundsControl/Visuals/TranslationHandles.cs:49]: Looks like the rotation issue is not a regression based on this comment: https://github.com/microsoft/MixedRealityToolkit-Unity/issues/9346#issuecomment-782428530

**Review by CDiaz-MS — APPROVED** [review]
Looks good to me:

BoundsControlHandlePR

**Pull request merged**